src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const property of action.keysToRemove) {
```

Comment:
[checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md) 3rd item
```suggestion
        for (const keyToRemove of action.keysToRemove) {
```